storage/CMakeLists.txt

```diff
@@ -56,16 +56,22 @@ google_cloud_cpp_add_common_options(storage_common_options)
 # Enable unit tests
 enable_testing()
 
-# Search for libcurl
+# Search for libcurl, in CMake 3.5 this does not define a target, but it does in later versions.
+# Define the target ourselves if it is missing.
```

Comment:
Still getting acquainted with CMake... maybe I missed it, but I don't see anything different in the CMake docs for find_package between v3.4 and v3.6.  Also, the comment below says that the find_package calls *do* define a target in CMake 3.5.  I'm just confused as to which versions do what, and why.   Are there docs or some explanation of this you can link?

Reply:
Just added a link, it is in development (use the `git-stage` pull down in the documentation). Generally, we are sticking to cmake-3.5 because it is easier for our users to find that one in the
platforms we support.